compute_tools/src/compute.rs

```diff
@@ -366,11 +366,29 @@ impl ComputeNode {
 
         let basebackup_cmd = match lsn {
             // HACK We don't use compression on first start (Lsn(0)) because there's no API for it
-            Lsn(0) => format!("basebackup {} {}", spec.tenant_id, spec.timeline_id),
-            _ => format!(
-                "basebackup {} {} {} --gzip",
-                spec.tenant_id, spec.timeline_id, lsn
-            ),
+            Lsn(0) => {
+                if spec.spec.mode != ComputeMode::Primary {
+                    format!(
+                        "basebackup {} {} --replica",
```

Comment:
This preferably shouldn't be done within the same PR as it will effectively block any normal compute releases (without cherry-picking) due to the hard requirement that new computes require new storage (iiuc)

I'd do it in 2 PRs
- extend PS API, then do storage release
- change computes

Reply:
Ok, I can split it into two PRs. But then there will be no tests win first PR.
I wonder if we can just enforce that storage is deployed prior to compute?